account-kit/rn-signer/example/src/screens/Home.tsx

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable import/extensions */
+import { RNAlchemySigner } from "@account-kit/react-native-signer";
+import type { User } from "@account-kit/signer";
+import { useEffect, useState } from "react";
+import {
+  View,
+  Text,
+  TextInput,
+  StyleSheet,
+  Linking,
+  TouchableOpacity,
+} from "react-native";
+import Config from "react-native-config";
+
+export default function HomeScreen() {
+  const [email, setEmail] = useState<string>("");
+  const [user, setUser] = useState<User | null>(null);
+  const signer = new RNAlchemySigner({
+    client: { connection: { apiKey: Config.API_KEY! } },
+  });
+
+  const handleUserAuth = ({ bundle }: { bundle: string }) => {
+    signer
+      .authenticate({
+        bundle,
+        type: "email",
+      })
+      .then(setUser)
+      .catch(console.error);
+  };
+
+  const handleIncomingURL = (event: { url: string }) => {
+    const regex = /[?&]([^=#]+)=([^&#]*)/g;
+
+    let params: Record<string, string> = {};
+    let match: RegExpExecArray | null;
+
+    while ((match = regex.exec(event.url))) {
+      if (match[1] && match[2]) {
+        params[match[1]] = match[2];
+      }
+    }
+
+    if (!params.bundle || !params.orgId) {
+      return;
+    }
+
+    handleUserAuth({
+      bundle: params.bundle ?? "",
+    });
+  };
+
+  useEffect(() => {
+    // get the user if already logged in
+    signer.getAuthDetails().then(setUser);
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'signer'. Either include it or remove the dependency array.
<!-- __reviewdog__:ChBjYTYzOWIzMTQ3MDMxZjE4EgZyZGpzb24= -->